Automattic/jetpack
# RNA: IDC setup fresh connection

On hold until #21162 and #21260 get merged.

#### Changes proposed in this Pull Request:
- make the "Create a fresh connection" button on the IDC screen create a new connection
- add the "IDC start fresh" method to the RNA API package
- make the "IDC start fresh" endpoint accept `redirect_uri` parameter

#### Jetpack product discussion
p9dueE-3kA-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Go to "Tools -> Connection Manager".
2. The "Safe Mode" screen should appear. Click "Create a fresh connection".
3. The button should display the loader animation, then you should get redirected to Calypso connection flow.
4. After clicking "Approve" you should get redirected back to "Tools -> Connection Manager".
